src/main/java/htsjdk/samtools/cram/CRAIEntry.java

```diff
@@ -33,6 +32,10 @@ public CRAIEntry(final int sequenceId,
                      final long containerStartByteOffset,
                      final int sliceByteOffset,
                      final int sliceByteSize) {
+        if (sequenceId == ReferenceContext.MULTIPLE_REFERENCE_ID) {
```

Comment:
Someday (not this PR) we can clamp this down even further; other negative values can be rejected.